proto/decider/v1/decider.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package decider.v1;
```

Comment:
Is the intent behind `v1` to map `awm-relayer` versions to `Decider` versions? If so, I'd prefer if we followed the pattern used by Avalanchego / subnet-evm where protocol version [compatibility](https://github.com/ava-labs/subnet-evm?tab=readme-ov-file#avalanchego-compatibility) is provided out-of-band. Otherwise, the codebase would continue to grow with each new version.

Reply:
the intent was just to follow the normal/standard behavior/conventions of buf+proto, at least as our starting point here. I stripped all of the custom lint rules out of the buf config file when copying it from avalanchego, and one of those was about needing to include the version in the service name.

in https://github.com/ava-labs/awm-relayer/pull/344/commits/3631b7bcf178e1b8e342434541fb84bd0b192ff5 I restored that linter rule and changed things to not include the version in the service/package names.